src/Lesson1/Task1.js

```diff
@@ -0,0 +1,13 @@
+// 1. Написать функцию loop, которая будет принимать параметры: times (значение по умолчанию = 0), 
+// callback (значение по умолчанию = null) и будет в цикле (times раз), вызывать функцию callback. 
+// Если функцию не передана, то цикл не должен отрабатывать ни разу. Покажите применение этой функции
+
+const loop = (times=0,callback=null) => {
+    for (let i = 0;i<times;i++){
```

Comment:
```suggestion
if (typeof callback === 'function') {
    for (let i = 0;i<times;i++){
```
"Если функцию не передана, то цикл не должен отрабатывать ни разу":)